src/content/en/fundamentals/performance/speed-tools/index.md

```diff
@@ -0,0 +1,334 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2018-02-26 #}
+{# wf_published_on: 2018-02-26 #}
+{# wf_blink_components: N/A #}
+
+<style>
```

Comment:
is all of this styling necessary? is it possible to use the existing DevSite styles?